projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -715,7 +715,16 @@ public static function enqueue_block_editor_assets() {
 				 *
 				 * @param bool false Enable the Paid Newsletters feature in the block editor context.
 				 */
-				'is_newsletter_feature_enabled' => self::is_newsletter_enabled(),
+				'is_newsletter_feature_enabled' => apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false ),
```

Comment:
```suggestion
				'is_newsletter_feature_enabled' => ( apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false ) && class_exists( '\Jetpack_Memberships' ) ),
```

Reply:
Fixed in a915233d7b4a78ad972aad2b6f3c27f971d37160